Modules/Quotas/src/Quotas.Domain/Aggregates/FileMetadata/FileMetadata.cs

```diff
@@ -1,8 +1,8 @@
 ﻿namespace Backbone.Modules.Quotas.Domain.Aggregates.FileMetadata;
 public class FileMetadata : ICreatedAt
 {
-    public string Id { get; set; }
+    public string? Id { get; set; }
```

Comment:
I checked the `Files.FileMetadata` table, you can see it in pgAdmin, and there you can see the columns corresponding to all of these four properties, therefore I don't think two of those should be nullable.

Reply:
👍